chevah/python-package
# [#4040] No more generic symlinks.

Scope
=====

we have start pushing the new build system in all project ... and those repos which were not yet updated, should be updated.

for this we should stop creating symlinks in the python package as they are no longer used.

With this, we should update the paver.sh with the latest version from brink ... to keep them up to date

Changes
=======

Have removed the code for generating symbolic links per OS/ARCH.
Have updated `paver.sh` from `brink` with related changes in `paver.conf` and `pavement.py` (updated used brink/twisted versions to latest).

**Drive-by change**:
  * got rid of the count to 16 while setting version per platform (code manually tested on the exotic platforms)
  * updated `get-pip.py` to latest version and set pip to 9.0.1.


How to try and test the changes
===============================

reviewers: @adiroiban 

Please review the changes.
Run `./paver.sh deps`.
Run `./chevah_build build` and notice there is no generic symlink in `dist/` anymore.
Run the automated tests, eg. https://chevah.com/buildbot/builders/python-package-gk-review/builds/52
